.circleci/config.yml

```diff
@@ -33,13 +33,19 @@ jobs:
               sleep 1
             done
             echo Failed waiting for Postgres && exit 1
+      - run:
+          name: Go Linter
+          command: |
+            GO111MODULE=auto go get -u golang.org/x/lint/golint
```

Comment:
Have a look at the gometalinter, you probably want to run at least `lint`, `vet`, and `fmt -s`.

Reply:
ah thanks! I'll add them too.